IyiKuyoro/Ride-My-Way-Frontend
# #159144479: Implement rides sorting

### What does this PR do?
Add rides sorting
#### Description of Task to be completed?
> - Add all rides origin to the from selector
> - Add all rides destination to the to selector
> - Filter the displayed rides based on the selected value
#### How should this be manually tested?
> - Clone the repository to your local  machine
> - Launch the index page and log in if you already have an account
> - If you don't have an account signup from the sign-up page Then login
> - See that the rides origins and destinations 
#### What are the relevant pivotal tracker stories?
#159144479
